schema/table.go

```diff
@@ -407,14 +382,6 @@ func (t *Table) filterDfs(parentMatched bool, include, exclude func(*Table) bool
 	return nil
 }
 
-func (t *Table) ValidateName() error {
```

Comment:
There's also a function to validate column names, but I kept it due to https://github.com/cloudquery/plugin-sdk/blob/3dbf2e978e2f650c281ccac342b931a33940c525/transformers/struct.go#L370